wpeverest/everest-forms
# Tweak  - Coupon into module

### All Submissions:



### Changes proposed in this Pull Request:

This PR introduces a new feature to the Coupon addon that allows users to generate coupon codes in bulk. Previously, the addon only supported single coupon code generation. This enhancement will streamline the coupon creation process, making it easier for users to generate multiple codes efficiently.

[PR DEPENDENCY](https://github.com/wpeverest/everest-forms-pro/pull/951)

Closes # .

### How to test the changes in this Pull Request:

1. Go to **Everest Forms Dashboard > Features > Enable Coupons Addons.**
2. Now go to **Coupons > Create New Coupon**
3. Pop up will appear to choose the coupon code generation type either **Single Coupon** or **Bulk Coupon Code.**
4. To create the **single coupon code** [refer to this docs](https://docs.everestforms.net/docs/coupons/#2-toc-title). 
5. To create the **bulk coupon code** follow following steps:
      -> Enter the Coupon Number, prefix, discount type, Start and End date, Select the Form for coupons to be applied.
6. Now Check whether the newly created or generated coupon code is working accordingly or not.

### Types of changes:




### Other information:




### Changelog entry

> Tweak  - Coupon into module
